diesel/src/deserialize.rs

```diff
@@ -166,7 +166,7 @@ where
 /// [`MysqlType`]: ../mysql/enum.MysqlType.html
 pub trait FromSql<A, DB: Backend>: Sized {
     /// See the trait documentation.
-    fn from_sql(bytes: Option<&DB::RawValue>) -> Result<Self>;
+    fn from_sql<'a>(bytes: Option<<DB::RawValue as FamilyLt<'a>>::Out>) -> Result<Self>;
```

Comment:
I think we should provide some type def for that type :wink:

```rust
type RawValue<'a, DB> = <DB::RawValue as FamilyLt<'a>>::Out;
```

Then use it as `RawValue<'a, DB>`.